python/ray/tune/tests/test_tune_relative_logdir.py

```diff
@@ -0,0 +1,133 @@
+# coding: utf-8
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import shutil
+import unittest
+
+import ray
+from ray import tune
+from ray.rllib.agents.registry import get_agent_class
+from ray.tune.registry import register_trainable
+
+
+class TuneRelativeLocalDirTest(unittest.TestCase):
+    @classmethod
+    def setUpClass(self):
+        self.algo = 'PG'
+        register_trainable(self.algo, get_agent_class(self.algo))
+        self.current_dir = os.path.abspath(".")
+        ray.init(num_cpus=1, num_gpus=0, local_mode=True)
+
+    @classmethod
+    def tearDownClass(self):
+        ray.shutdown()
+
+    def tearDown(self):
+        os.chdir(self.current_dir)
+
+    def testDottedRelativePath(self):
+        local_dir = "test_dotted_relative_local_dir"
+        local_dir = os.path.join(self.current_dir, local_dir)
+
+        tune.run(
+            self.algo,
+            name="TuneDottedRelavtieLocalDirTest",
```

Comment:
```suggestion
            name="TuneDottedRelativeLocalDirTest",
```
Same thing in some other places below.